scipy/stats/_unuran/unuran_wrapper.pyx.templ

```diff
@@ -969,6 +969,185 @@ cdef class TransformedDensityRejection(Method):
         return np.asarray(out).reshape(oshape)[()]
 
 
+cdef class SimpleRatioUniforms(Method):
```

Comment:
A side note, this file is getting large. Maybe we can split it at some point.

Reply:
You are right. I can submit a maintenance patch mid-december that splits invention methods, continuous methods, and discrete methods in their own file and improves the readability of the code.